info_file.txt

```diff
@@ -0,0 +1,84 @@
+netcdf dataset_2 {
```

Comment:
@yvanlebras the two news files, for what are they good for? They are not used in this PR afaik.

Reply:
hmmm really sorry, don't know how these 2 files (test-data from the nectdf GH branch) are arrive here ... AGAIN.... TTHHHHAAAAANNK   YYOUUUUUU you and your eagle eyes ;)